.gitignore

```diff
@@ -5,6 +5,7 @@ debug.*
 .vscode/
 .idea
 .keep
+.jekyll-cache/
```

Comment:
`j `before `k`

Reply:
this is not at all alphabetical (see `vscode` just above)

Reply:
Even if it isn't alphabetical, it seems like it's in the wrong section (it should be around other `.<dirname>/` entry which is `.vscode/`). Maybe we just need a quick cleanup of this file?

Reply:
ok, will take a stab at it